src/web/core/src/main/java/org/geoserver/web/data/layer/SQLViewAbstractPage.java

```diff
@@ -305,7 +307,16 @@ protected Component getComponentForProperty(
                     public void onSubmit() {
                         onSave();
                     }
-                });
+                }.setVisible(newView));
+
+        form.add(
+                new SubmitLink("back") {
+                    @Override
+                    public void onSubmit() {
+                        onSave();
```

Comment:
I do not understand what this `back` button is doing? the implementation looks identical to `save` above?

Reply:
It actually does not save anything, just updates the featuretype in memory, does not save it in the catalog.
Make sense as this is a sub-page of the layer page. So it's "back" to the layer page, you have to save there.